vulnfeeds/cmd/cperepos/gen_cperepos_map

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#
+# Maintain a regularly generated CPE to repo mapping in a GCS bucket.
+#
+# Inputs:
+# * A local work directory
+# * A GCS bucket name + path for Debian copyright metadata
+# * A GCS bucket name + path for the resulting map file
+
+# Setting BE_VERBOSE to an empty string or null value suppresses silencing of
+# commands
+
+mkdir -p "${WORK_DIR}" || true
+
+curl ${BE_VERBOSE="--silent"} \
+  https://nvd.nist.gov/feeds/xml/cpe/dictionary/official-cpe-dictionary_v2.3.xml.gz \
+  | gzip --decompress --output "${WORK_DIR}/official-cpe-dictionary_v2.3.xml"
+
+MAYBE_USE_DEBIAN_COPYRIGHT_METADATA=""
+if [[ -n "${DEBIAN_COPYRIGHT_GCS_PATH}" ]]; then
+  gsutil ${BE_VERBOSE="-q"} -m rsync -d -r "${DEBIAN_COPYRIGHT_GCS_PATH}" "${WORK_DIR}"
+  MAYBE_USE_DEBIAN_COPYRIGHT_METADATA="--debian_metadata_path ${WORK_DIR}/debian_copyright/metadata.ftp-master.debian.org"
+fi
+
+/src/cperepos \
+  --cpe_dictionary "${WORK_DIR}/official-cpe-dictionary_v2.3.xml" \
+  ${MAYBE_USE_DEBIAN_COPYRIGHT_METADATA} \
+  --output_dir "${WORK_DIR}"
+
+gsutil ${BE_VERBOSE="-q"} rsync -d -r "${WORK_DIR}/cpe_product_to_repo.json" "${CPEREPO_GCS_PATH}"
```

Comment:
Add `-c` after `rsync` to only upload changed files, this preserves created_time on the bucket entries.